ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -65,6 +66,12 @@ export default function TokenList({
     contractExchangeRates,
   ]);
 
+  useEffect(() => {
+    if (!loading) {
+      endTrace({ name: TraceName.AccountOverviewAssetListTab });
+    }
+  }, [loading]);
```

Comment:
From https://react.dev/learn/you-might-not-need-an-effect#recap
> If you can calculate something during render, you don’t need an Effect.

`useEffect` to run code "when this variable changes" is done **A LOT**, but it's actually an anti-pattern

Reply:
```suggestion
  if (!loading) {
    endTrace({ name: TraceName.AccountOverviewAssetListTab });
  }
```